pages/dkp/kommander/1.4/projects/platform-services/platform-services-catalog/kudo-spark/index.md

```diff
@@ -0,0 +1,518 @@
+
+The current parameter set can be retrieved using the kubectl command in conjunction with two additional tools:
+- [jq](https://stedolan.github.io/jq)
+- [yq](https://mikefarah.gitbook.io/yq)
+
+To retrive the current parameters, issue the following command in the terminal with appropriate `INSTANCE` value set:
+
+```
+INSTANCE=spark-instance;
+kubectl get instances -o json | jq ".items[] | select(.metadata.name == \"$INSTANCE\") | .spec.parameters" | yq eval -P > spark-params.yml
+```
+
+The above command generates a file called `spark-params.yml` with the current values of all the parameters in effect for the `spark` operator instance.
+
+For more information, see [Advanced Configuration](https://github.com/mesosphere/kudo-spark-operator/blob/master/kudo-spark-operator/docs/latest/advanced-configuration.md).
+
+### Updating Operator Parameters
+
+Parameters can be updated using arguments to the KUDO CLI.
+
+**Example**: Enabling Spark History Server
+- Enabling [Spark History Server](https://github.com/mesosphere/kudo-spark-operator/blob/master/kudo-spark-operator/docs/latest/history-server.md) using the KUDO CLI:
+```
+kubectl kudo update --instance spark-instance -p enableHistoryServer=true -p historyServerFsLogDirectory=file:///tmp/ -n spark
+```
+- Monitor the KUDO Spark deployment plan:
+```
+kubectl kudo plan status --instance spark-instance -n spark
+```
+- Wait for the deployment plan to have a status of `COMPLETE`
+
+When the deployment plan is `COMPLETE` there should be Spark History Server pod running:
+```
+kubectl get pod -l app.kubernetes.io/name=spark-history-server -n spark
+NAME                                             READY   STATUS    RESTARTS   AGE
+spark-instance-history-server-68f5645c57-5h2g4   1/1     Running   0          31s
+```
+
+**Example**: Updating parameters providing YAML file with parameters:
+
+To update multiple parameters at once, it is recommended to submit the updated parameters using the KUDO CLI.
+
+See [Available Parameters](#available-operator-parameters) to get the full list of current parameters as a file.
+
+Apply the desired updates in `spark-params.yml` using the KUDO CLI:
+```
+kubectl kudo update --instance=spark-instance -P spark-params.yml -n spark
+```
+Wait for the deployment plan to `COMPLETE`.
+
+### Upgrades
+
+**Example**: Upgrade KUDO Spark Operator from `v3.0.0-1.0.1` to `v3.0.0-1.1.0`
+
+When upgrading, you should understand the mapping between Spark versions and operator versions.
+Wait and monitor the deployment plan to become `COMPLETE`.
+
+<p class="message--note"><strong>NOTE: </strong>Spark Operator version doesn't necessarily have to match the Apache Spark version to run Spark Applications built with that version. For instance, it is possible to submit Spark 2.4.5 Application using the operator version 3.0.0-1.1.0.</p>
```

Comment:
I think this is something @russurquhart1 would say: 

```suggestion
<p class="message--note"><strong>NOTE: </strong>Spark Operator version doesn't have to match the Apache Spark version to run Spark Applications built with that version. For instance, it is possible to submit Spark 2.4.5 Application using the operator version 3.0.0-1.1.0.</p>
```

Reply:
applied